kaggle-classification/keras_trainer/model.py

```diff
@@ -42,7 +43,7 @@
     learning_rate=0.00005,
     dropout_rate=0.5,
     batch_size=128,
-    epochs=1,
+    epochs=10,
```

Comment:
Is there a reason this changed back to 10?

Reply:
No, will change back to 1